modules/support_ticket/config/install/system.action.support_ticket_delete_action.yml

```diff
@@ -0,0 +1,9 @@
+id: support_ticket_delete_action
+label: 'Delete support ticket'
```

Comment:
As step 2, should add configurable labels, or rather:

Start with 'Delete entity' here, then override with 'Delete %bundle_name' later.